ZuluContent/SkillHandlers/ArmsLore.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Threading.Tasks;
+using Scripts.Zulu.Engines.Classes;
+using Server.Items;
+using Server.Mobiles;
+using Server.Targeting;
+using ZuluContent.Zulu.Skills;
+
+namespace Server.SkillHandlers
+{
+    public class ArmsLore : BaseSkillHandler
+    {
+        public override SkillName Skill { get; } = SkillName.ArmsLore;
+
+        private static readonly TargetOptions TargetOptions = new()
+        {
+            Range = 2,
+            AllowNonLocal = true,
+        };
+
+        public override async Task<TimeSpan> OnUse(Mobile from)
```

Comment:
Kept Arms Lore pretty much the same as it came from RunUO.T2A, the differences are pretty minor. Whole skill needs a rework really.

Reply:
ok cool